_build/partials/sandbox_about.hbs

```diff
@@ -10,12 +10,30 @@
         <p>
           <img src="{{rootUrl}}img/external/world.png" alt="Worldstrat locations" width=100% />
         </p> 
-        <p class="legend" align="right">Map data from <a href="https://www.openstreetmap.org/copyright" target="_blank">OpenStreetMap</a>.</p>
+        <p class="legend" align="right">Available Sandbox Data locations. Basemap from <a href="https://www.openstreetmap.org/copyright" target="_blank">OpenStreetMap</a>.</p>
     </div>
     <hr>
     <p>Planet's data, with its dense time series and systematic availability, makes it a perfect input to machine learning models. Sandbox Data serves as a rich and diverse set of data collections, making it possible for data scientists to build, test and validate their machine learning models without huge investment in data subscriptions, even prior to validating the business model.</p>
-
-    <p> Discover how to use <a href="https://devext.sinergise.com/eo-browser/feature-sko/?tutorialIdToShow=PSD_TUTORIAL" target="_blank">Planet Sandbox Data in EO Browser</a>.</p>
-
+    <a href="https://apps.sentinel-hub.com/eo-browser/?tutorialIdToShow=PSD_TUTORIAL" target="_blank">
+      <div class="explore-sbd-btn">
+        <h3>
+          Explore Sandbox Data via EO Browser →
+        </h3>
+        <h4>
+          Discover the easy-to-use theme and selected higlights.*
+        </h4>
+      </div>
+    </a>
+    <a href="https://apps.sentinel-hub.com/eo-browser/?tutorialIdToShow=PSD_TUTORIAL" target="_blank">
```

Comment:
This still needs to be updated once we have the final link.

Reply:
Talked with Dino about this yes, best to point all links to production and eventually it's going to work

Reply:
Sorry, I made a mistake when checking this and wrote nonsense. This link needs to be updated to point to github repo of jupyter notebooks

Reply:
Yes, Dino said, the link is not yet there and we should just merge this and update the link later.